reader/reader.go

```diff
@@ -22,19 +23,22 @@ func readFile(f io.Reader) ([]byte, error) {
 		input = append(input, line...)
 	}
 	if err != io.EOF {
-		return nil, fmt.Errorf("error reading file: %s", err.Error())
+		return nil, fmt.Errorf("error reading input: %s", err.Error())
+	}
+	if len(input) == 0 {
+		return nil, errors.New("no input data; expected input via a non-empty file or via STDIN")
 	}
 	return input, nil
 }
 
-func CreateReader(stdin *os.File, args []string) (Reader, error) {
-	stat, _ := stdin.Stat()
-	if (stat.Mode() & os.ModeCharDevice) == 0 {
-		return NewStdinReader(), nil
```

Comment:
☝️ Previously, in some environments -- like GitHub Actions Ubuntu runners -- `tf-summarize` enters this codepath, even when an explicit filename argument has been provided.

Reply:
Got it. Resolving